src/components/Organization/ViewOrganization/index.js

```diff
@@ -235,6 +249,7 @@ const ViewOrganization = () => {
                       justifyContent="flex-start"
                       direction="column"
                       spacing={5}
+                      className={classes.Activity}
```

Comment:
do not use capital letters to name a variable. shall we change this? (unless it is a class name :D)

Reply:
oh, suree